src/plugins/english/relibrary.ts

```diff
@@ -164,7 +164,19 @@ class ReLibraryPlugin implements Plugin.PluginBase {
   name = 'Re:Library';
   icon = 'src/en/relibrary/icon.png';
   site = 'https://re-library.com';
-  version = '1.0.0';
+  version = '1.1.0';
+
+  imageRequestInit: Plugin.ImageRequestInit = {
+    headers: {
+      Referer: ReLibraryPlugin.site,
+    },
+  };
+
+  private static trim_slashes(url: string): string {
+    while (url.startsWith('/')) url = url.slice(1);
+    while (url.endsWith('/')) url = url.slice(0, url.length - 1);
+    return url;
+  }
```

Comment:
be careful, after this changes, novels in popular and search would be treated as new novels (though they were installed) because the path is not same as in the database